specification/iotoperations/IoTOperations.Management/models/auth.tsp

```diff
@@ -20,7 +20,7 @@ model DataflowEndpointAuthenticationUserAssignedManagedIdentity {
   clientId: string;
 
   @doc("Resource identifier (application ID URI) of the resource, affixed with the .default suffix.")
-  scope: string;
+  audience?: string;
```

Comment:
John Lian has asked to keep this as scope

Reply:
Is this just for User assigned idenity?